api/lib/services/storage/storage-types/index.ts

```diff
@@ -1,4 +1,4 @@
-import Storage from '@google-cloud/storage';
+import storage from '@google-cloud/storage';
```

Comment:
This is a breaking change. Please change this back to `Storage` as some of the dependent code will not work otherwise.